packages/components/src/components/dropdown/items.jsx

```diff
@@ -18,7 +18,7 @@ const Item = ({ onKeyPressed, value, item, handleSelect, nodes, has_symbol, is_a
     React.useEffect(() => {
         const removeListeners = () => {
             nodes.delete(item.value, item_ref.current);
-            item_ref.current.removeEventListener('keydown', onKeyPressed);
+            item_ref.current?.removeEventListener('keydown', onKeyPressed);
```

Comment:
Error `Cannot read properties of null (reading 'removeEventListener')` appears on all dropdown, so I've made decision to fix it too.